sway/input/seat.c

```diff
@@ -989,6 +993,10 @@ void seat_begin_down(struct sway_seat *seat, struct sway_container *con,
 	seat->op_ref_con_lx = sx;
 	seat->op_ref_con_ly = sy;
 	seat->op_moved = false;
+
+	if (con && !config->raise_floating) {
```

Comment:
This condition looks wrong, but I think it's right. If you've moved the mouse over the floating container and it's given focus but hasn't raised because `raise_floating` is disabled, this part of the code raises it on click. Maybe add a comment explaining why this is here.

Reply:
I amended the second commit with a comment.